src/foam/nanos/script/Script.js

```diff
@@ -324,6 +324,12 @@ foam.CLASS({
         return getId();
       `
     },
+    function clone() {
```

Comment:
these functions don't make sense - if we're cloning, then why are we just returning `this` immediately? they also don't seem to be used by this PR...

Reply:
@petenanopay these override the defaults in FObject.java which do copy.  We are explicitly not copying in this case.

Reply:
@hchoura Are these two clone() functions required to fix the problem? If so, then the issue is that the Controller is now clone()-ing, which is breaking this feature. We should fix it there and leave out these methods. These were just to test if the problem was clone related.

Reply:
I retest it. We can delete them.